src/Sylius/Bundle/ApiBundle/Resources/config/api_platform/resources/AdminResetPassword.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Sylius Sp. z o.o.
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<resources xmlns="https://api-platform.com/schema/metadata/resources-3.0"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="https://api-platform.com/schema/metadata/resources-3.0 https://api-platform.com/schema/metadata/resources-3.0.xsd"
+>
+    <resource
+        class="Sylius\Bundle\CoreBundle\Message\Admin\Account\ResetPassword"
+        shortName="AdminResetPasswordRequest"
```

Comment:
A bit is mismatch in resource naming, it's `AdminResetPassword` or `ResetPassword` everywhere, but here it's `AdminResetPasswordRequest`.

Would be cool to sync terminology somehow.